core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -99,6 +99,10 @@ class SparkEnv (
 
   private[spark] var executorBackend: Option[ExecutorBackend] = None
 
+  private[spark] var cachedArrowBatchServerPort: Option[Int] = None
+
+  private[spark] var cachedArrowBatchServerSecret: Option[String] = None
```

Comment:
shall we have a single function to return a tuple2?

Reply:
Pyspark code will call these functions, tuple type return value makes python code unclear (we have to write `._1()` `._2()` in python code